### PR TITLE
run: extra cleanup needed for test selection

### PR DESCRIPTION
**Pull request by mchalski**
the final if probably had no purpose, and prevented from
running SPECIFIC_INTEGRATION_TESTs.

changelog: none

Signed-off-by: Marcin Chalczynski <m.chalczynski@gmail.com>

### CODE REVIEW

File: tests/run.sh [hunk 1]
```diff
@@ -181,17 +181,8 @@ if ! pip2 list|grep -e pytest-html >/dev/null 2>&1; then
...
     SPECIFIC_INTEGRATION_TEST_ARG="-k $SPECIFIC_INTEGRATION_TEST"
 fi
 
-if [ $# -eq 0 ]; then
-    py.test $XDIST_ARGS $MAX_FAIL_ARG -s --verbose --junitxml=results.xml $HTML_REPORT --runfast --runslow $UPGRADE_TEST_ARG $SPECIFIC_INTEGRATION_TEST_ARG $DEFAULT_TESTS
-    exit $?
-fi
-
-python -m pytest $XDIST_ARGS $MAX_FAIL_ARG -s --verbose --junitxml=results.xml $HTML_REPORT $pass_args $DEFAULT_TESTS
+python -m pytest $XDIST_ARGS $MAX_FAIL_ARG -s --verbose --junitxml=results.xml $HTML_REPORT $pass_args $SPECIFIC_INTEGRATION_TEST_ARG $DEFAULT_TESTS
```
Comment (lluiscampos) [inline tests/run.sh]: Nit-pick: I had [this commit](https://github.com/mendersoftware/integration/pull/670/commits/d85082bd6db3024d8b76bdc1773274dbc5a316e2) in my PR, can you use `python2 -m pytest` and I discard my change?

Comment (mchalski) [reply tests/run.sh]: sure, hold on

Comment (mchalski) [reply tests/run.sh]: done

**Comment by mchalski** [comment]
i'm running a test pipeline with mender-qa/290 , this one and a dummy deployments-enterprise PR - just to see if it has any hope of working.

**Pull request merged**